ironfish-cli/src/utils/multisig.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { FileSystem, YupUtils } from '@ironfish/sdk'
+import * as yup from 'yup'
+
+export type MultisigTransactionOptions = {
+  identity: string[]
+  unsignedTransaction: string
+  commitment: string[]
+  signingPackage: string
+  signatureShare: string[]
+}
+
+export const MultisigTransactionOptionsSchema: yup.ObjectSchema<
+  Partial<MultisigTransactionOptions>
+> = yup
+  .object({
+    identity: yup.array().of(yup.string().defined()),
+    unsignedTransaction: yup.string(),
+    commitment: yup.array().of(yup.string().defined()),
+    signingPackage: yup.string(),
+    signatureShare: yup.array().of(yup.string().defined()),
+  })
+  .defined()
+
+async function load(
+  files: FileSystem,
+  path?: string,
+): Promise<Partial<MultisigTransactionOptions>> {
+  if (path === undefined) {
+    return {}
+  }
+
+  const data = (await files.readFile(files.resolve(path))).trim()
+
+  const { error, result } = await YupUtils.tryValidate(MultisigTransactionOptionsSchema, data)
+
+  if (error) {
+    throw error
+  }
+
+  return result
+}
+
+type MultisigTransactionFlags = {
```

Comment:
Do you mean fields that correspond to other flags not listed here, like `confirm`?

I was thinking of this JSON file more as configuration of the whole transaction rather than a specific command